src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/BehandlingController.kt

```diff
@@ -80,7 +83,7 @@ class BehandlingController(
         tilgangService.validerHarSaksbehandlerrolle()
         val henlagtBehandling = henleggService.henleggBehandling(behandlingId, henlagt)
         val fagsak: Fagsak = fagsakService.hentFagsak(henlagtBehandling.fagsakId)
-        return henlagtBehandling.tilDto(fagsak.stønadstype)
+        return henlagtBehandling.tilDto(fagsak.stønadstype, fagsak.fagsakPersonId)
```

Comment:
For å unngå den "rare" tildto på behandling, burde vi kanskje prøve å kun bruke saksbehandling og då henlegge og sen hente saksbehandlingen 👀 
```kotlin
        henleggService.henleggBehandling(behandlingId, henlagt)
        return behandlingService.hentSaksbehandling(behandlingId).tilDto()
```

Et til alternativt er at tilDto() setter sammen behandling + fagsak, og at man sender inn fagsak til `tilDto`? 👀

Reply:
Ja, tenkte litt på å fjerne `behandling.TilDto()` men ble litt utafor scope for dette.